wizard.py

```diff
@@ -362,7 +362,7 @@ def ajax_flash_messages(response):
   show_messages(repsonse.messages).
   """
 
-  if (request.is_xhr and
+  if (request.headers.get("X-Requested-With") == "XMLHttpRequest"  and
```

Comment:
IIUC you are re-implementing the check that the deprecated `request.is_xhr` used to perform, right?

Did you have a chance to check if the jquery ajax call we use still sets that header, so that our `ajax_flash_messages` hook still works?

Reply:
Thanks again for the review @lukpueh ! I am a little bit confused about the `ajax_flash_messages hook` and I am not sure what a hook does. Could you please explain it a bit more?

Reply:
Thanks a ton for the review @lukpueh !!
I checked the `ajax_flash_messages` hook and something is not right.
For example, when I upload the functionary's public key in the functionaries page, the flash messages show up only after I manually refresh the page. When I try to remove the functionary that already has a pubkey, I also have to refresh the page to see the change and the flash message. I am not sure if there is something wrong with the ajax call or the `ajax_flash_messages` hook, but I will try to find up why this happened.

Reply:
As a follow-up, I will work on [rule generation](https://github.com/in-toto/layout-web-tool/issues/48#issuecomment-671955671) first, which seems to have a higher priority.

Reply:
I found `ajax_flash_messages` working again.

I check the jquery ajax call with the browser's network console, and it does include the `X-Requested-With` header with the value `XMLHttpRequest`. So I believe there is no problem with the code here. I doubt that it failed previously is because my Virtual Machine didn't run the WSGI server correctly.

Reply:
I'll merge then. Thanks for the updates!